src/main/java/org/jenkinsci/plugins/docker/commons/tools/DockerToolInstaller.java

```diff
@@ -162,7 +163,7 @@ public Object invoke(File f, VirtualChannel channel) throws IOException, Interru
         return install;
     }
 
-    @Extension
+    @Extension @Symbol("dockerIoInstaller")
```

Comment:
Yeah, I'm gonna let this sit for a day or so for me to think about - I may just yank most of these `@Symbol`s since it's really hard to figure out what they should actually be called...